patches/minecraft/net/minecraft/world/item/Rarity.java.patch

```diff
@@ -9,13 +9,31 @@
     COMMON(ChatFormatting.WHITE),
     UNCOMMON(ChatFormatting.YELLOW),
     RARE(ChatFormatting.AQUA),
-@@ -12,5 +_,9 @@
+    EPIC(ChatFormatting.LIGHT_PURPLE);
+ 
++   @Deprecated // Forge: Use getStyleModifier
```

Comment:
This is functionally different from a `@deprecated` javadoc, which would be preferred.